content/docs/guides/hedgedoc.md

```diff
@@ -0,0 +1,144 @@
+# HedgeDoc
+
+ Use Pomerium to secure access to HedgeDoc by providing identity and context. 
+
+ ## What is HedgeDoc?
+
+[HedgeDoc](https://hedgedoc.org/) is a collaborative, web-based Markdown editor that allows you to create notes, graphs, and diagrams in your browser. 
+
+You can share a link to your document with other members of your organization so you can collaborate in real time.
+
+ ## Integrate Pomerium with HedgeDoc
+ 
+ HedgeDoc is open-source, self-hosted software that allows you to configure authentication and authorization with a [reverse proxy](https://docs.hedgedoc.org/guides/reverse-proxy/). 
+ 
+ You can configure Pomerium so that it authenticates and authorizes users against an Identity Provider (IdP) before users can access HedgeDoc's services.
+
+The flow looks like this: 
+
+1. User requests to access the secured app
+2. Pomerium proxy identifies user
+3. Pomerium proxy checks if user is authorized to access routes defined in the authorization [`policy`](https://www.pomerium.com/docs/reference/routes/policy)
+4. If authorized, Pomerium proxy authenticates user against IdP
+5. After IdP establishes user's identity, Pomerium proxy checks permissions against the secured app
+6. User can make authorized requests in the secured app
+
+![Pomerium IAP flow](../img/pom-auth-flow.png)
+
+### Set up your environment
+This guide uses Docker containers to create the minimal development environment required to run Pomerium with another service. Visit the [Pomerium using Docker](https://www.pomerium.com/docs/install/quickstart) quickstart for more information.
+
+To complete this guide, you need: 
+- [Docker](https://www.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- A running Pomerium instance
+- A [preconfigured IdP](https://www.pomerium.com/docs/identity-providers)
+
+> **Note:** This guide uses [GitHub](https://www.pomerium.com/docs/identity-providers/github) as the preconfigured IdP.
+
+### Pomerium configuration
+
+Your Pomerium configuration file must include a [Cookie Secret](https://www.pomerium.com/docs/reference/cookie-secret) and a [Secret Key](https://www.pomerium.com/docs/reference/signing-key). 
+
+To create a Cookie Secret, run the following command in your terminal: 
+```bash
+head -c32 /dev/urandom | base64
+```
+
+To create a Secret Key, run the following command: 
+```bash
+cat ec_private.pem | base64
+```
+
+>**Note:** This guide assumes you've generated self-signed wildcard certificates. Check [Certificates](https://www.pomerium.com/docs/topics/certificates#self-signed-wildcard-certificate) for more information.
+
+Place your `cookie_secret` and `secret_key` in your `config.yaml` file: 
+
+```yaml
+cookie_secret: <cookie secret>
+signing_key: <signing key>
+```
+
+Lastly, you must enable your Pomerium proxy to enable [Websocket Connections](https://www.pomerium.com/docs/reference/routes/websocket-connections):
+
+```yaml
+allow_websockets: true
```

Comment:
Ideally, we could have a minimal but complete config.yaml that includes a route and policy pointing to hedgedoc here.